scripts/mock_data/supplementary_dataset_single_item_response.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
I think filename should include `metadata` to signify what this is used for. Same for the one below.